### DISCUSSION TITLE
Save and load the base layer drawn on

### DISCUSSION THREAD

**Discussion by problemz** [geoman-io/leaflet-geoman#979] [Questions & Help] [answered]
Hello! I have a project with many base layers and want to save the drawn items on every base layer and when loading them again only show them on the correct base layer. How can i get the "drawn on base layer x" and load it again?

**Comment by Falke-Design** [comment]
I think your base layers are L.LayerGroups or L.FeatureGroups. So then you can simply call `yourbaserlayer.getLayers()` and to add them again to the base layer `yourbaserlayer.addLayer(yourLayer)`. 

If you have other layers in the group then the drawn one too, you can check if the layers are drawn by Geoman: `yourLayer._drawnByGeoman === true`

**Comment by problemz** [comment]
I dont know how to export the value to geojson and then import it back .. thats my current testsite: https://fastcrow.run/e.html .. if you click on save on the sidebar you see that i get the drawn items per baselayer .. but how can i export a value to reimport them with a check? (if value= hod -> add to "hod_draw" for example .. sorry im very bad with this stuff.

**Reply by Falke-Design** [reply]
I will help you but you need to give me to complete project (i don't need the tiles), so i can edit it.

**Comment by Falke-Design** [comment, marked as answer]
Solved in a meeting
